src/mcts/search.h

```diff
@@ -277,6 +281,8 @@ class SearchWorker {
   };
 
   NodeToProcess PickNodeToExtend(int collision_limit);
+  void EvalPosition(Node* node, MoveList& legal_moves, const ChessBoard& board,
```

Comment:
Non-const references are not allowed by code style. As it's function output, use std::tuple<> return parameter instead?